### PR TITLE
Add Debian repo instructions and link to AUR

### LINKED ISSUE #44: Debian package repository
I'm going to set up a simple Debian package repository for Jellyfin. While eventually the goal should be to get ourselves included in upstream Debian, this will help make installing the server easy for users today, to avoid them having to build it themselves.

### PR DESCRIPTION
**Pull request by joshuaboniface**
Closes #44

### CODE REVIEW

**Comment by joshuaboniface** [comment]
Actually, I should add a bit about Docker Hub too.

File: README.md [hunk 1]
```diff
@@ -13,15 +13,51 @@ While our first priority is a stable build, we will eventually add features that
...
+A package repository is available at https://repo.jellyfin.org. To use it:
+
+0. Install the `dotnet-runtime-2.1` package via [Microsoft's repositories](https://dotnet.microsoft.com/download/dotnet-core/2.1).
+0. Import the GPG signing key (signed by Joshua):
+    ```
+    wget -O - https://repo.jellyfin.org/debian/jellyfin-signing-key-joshua.gpg.key | sudo apt-key add -
+    ```
+0. Add an entry to `/etc/sources.list.d/jellyfin.list`:
+    ```
+    echo "deb https://repo.jellyfin.org/debian $( grep -Ewo -m1 --color=none 'jessie|stretch|buster' /etc/os-release || echo buster ) main" | sudo tee /etc/apt/sources.list.d/jellyfin.list
```
Comment (z3ntu) [inline README.md]: This command only works on Debian but the section is titled "Debian/Ubuntu"

File: README.md [hunk 2]
```diff
@@ -13,15 +13,51 @@ While our first priority is a stable build, we will eventually add features that
...
+0. Install Jellyfin:
+    ```
+    sudo apt install jellyfin
+    ```
+
+## Building Jellyfin packages from source
 
 Jellyfin seeks to integrate build facilities for any desired packaging format. Instructions for the various formats can be found below.
 
+NOTE: When building from source, it is strongly advised to clone the full Git repository, rather than using a `.zip`/`.tar` archive.
```
Comment (z3ntu) [inline README.md]: Why? The submodules are gone so it shouldn't be a problem

Comment (joshuaboniface) [reply README.md]: We may add more in the future.

File: README.md (hunk 1 above)
Comment (joshuaboniface) [reply README.md]: I'm curious how it fails on Ubuntu - it should work the same.

Comment (z3ntu) [reply README.md]: Well Ubuntu users would only get the Debian buster packages and not the ones matching their Ubuntu Debian-base

File: README.md (hunk 2 above)
Comment (z3ntu) [reply README.md]: Then I would reword it to something like including the words "not supported"

File: README.md (hunk 1 above)
Comment (joshuaboniface) [reply README.md]: Yes that should be fine because there's no distro-specific parts to it. Really there isnt even a reason to separate the Debian versions - its the same package, I just did that for consistency.

Comment (z3ntu) [reply README.md]: Okay it's fine then

File: README.md [hunk 3]
```diff
@@ -13,15 +13,51 @@ While our first priority is a stable build, we will eventually add features that
...
+
 ### Debian/Ubuntu
 
 Debian build facilities are integrated into the repo at `debian/`.
 
-1. Install the `dotnet-sdk-2.1` package via [Microsoft's repositories](https://dotnet.microsoft.com/download/linux-package-manager/debian9/sdk-2.1.500).
+1. Install the `dotnet-sdk-2.1` package via [Microsoft's repositories](https://dotnet.microsoft.com/download/dotnet-core/2.1).
 2. Run `dpkg-buildpackage -us -uc -jX`, where X is your core count.
```
Comment (JustAMan) [inline README.md]: I think we can drop "-jX" part as it is not passed anywhere (at least not passed to `dotnet publish`).

Also might need a note regarding build as regular user - in my experience `dotnet publish` when run as regular user from `fakeroot` goes nuts. Or better solve that problem somehow...

Comment (joshuaboniface) [reply README.md]: Good point on the threads.

I've been running it in fakeroot without issue, weird.

Comment (JustAMan) [reply README.md]: Then it might be due to me running that in unprivileged lxc container... I'll try to drop a note about that later in readme.

File: README.md (hunk 2 above)
Comment (joshuaboniface) [reply README.md]: Good idea, I was trying to give tthat impression without saying as much, but lets be direct.

File: README.md [hunk 4]
```diff
@@ -13,15 +13,51 @@ While our first priority is a stable build, we will eventually add features that
...
+### Debian/Ubuntu
+
+A package repository is available at https://repo.jellyfin.org. To use it:
+
+0. Install the `dotnet-runtime-2.1` package via [Microsoft's repositories](https://dotnet.microsoft.com/download/dotnet-core/2.1).
+0. Import the GPG signing key (signed by Joshua):
+    ```
+    wget -O - https://repo.jellyfin.org/debian/jellyfin-signing-key-joshua.gpg.key | sudo apt-key add -
+    ```
+0. Add an entry to `/etc/sources.list.d/jellyfin.list`:
```
Comment (BnMcG) [inline README.md]: At least on Ubuntu, I believe this should be `/etc/apt/sources.list.d/jellyfin.list`

Comment (joshuaboniface) [reply README.md]: It is for Debian too - typo on my part. Fixed in an upcoming commit.
